Network Design Guide/readme.md

```diff
@@ -1,14 +1,14 @@
 # Azure VMware Solution Landing Zone Accelerator Network Design Guide
 This guide covers network design for Azure VMWare Solution. It encompasses four design areas, summarized below.
 
-1. **Connectivity with on-premises datacenters**. Connectivity between Azure VMware Solution private clouds and on-prem sites supports a broad set of use cases, such as HCX migrations, hybrid applications, remote vSphere or NSX-T Data Center administration. Azure VMware Solution supports multiple options for hybrid connectivity, including Azure ExpressRoute circuits and internet-based IPSec VPNs. 
+1. **Connectivity with on-premises datacenters**. Connectivity between Azure VMware Solution private clouds and on-premises sites supports a broad set of use cases, such as HCX migrations, hybrid applications, remote vCenter or NSX-T Manager administration. Azure VMware Solution supports multiple options for hybrid connectivity, including Azure ExpressRoute circuits and internet-based IPSec VPNs. 
 
-2. **Connectivity with Azure Virtual Networks**. Azure VMware Solution runs on bare-metal VMware VSphere clusters that can be connected to native Azure Virtual Networks (VNets) through Azure ExpressRoute. ExpressRoute connections between Azure VMware Solution private clouds and Azure VNET enable building applications that span the two environments or using “jump-box” virtual machines in Azure to log into vCenter Server and NSX-T Management console for administration purposes. 
+2. **Connectivity with Azure Virtual Networks**. Azure VMware Solution runs on bare-metal VMware vSphere clusters that can be connected to native Azure Virtual Networks (VNets) through Azure ExpressRoute. ExpressRoute connection between Azure VMware Solution private cloud and Azure VNet enables building applications that span the two environments or using “jump-box” virtual machines in Azure IaaS to log into vCenter Server and NSX-T Manager console for administration purposes. 
```

Comment:
There is definitely a typo here, but it is the missing "s" after VNet. The sentence should read "ExpressRoute connections between Azure VMware Solution private clouds and Azure VNETs enable building applications that span the two environments or using “jump-box” virtual machines....."

Reply:
Yes, now it makes more sense! Thanks @fguerri !